pandas/tests/frame/test_timeseries.py

```diff
@@ -57,6 +57,22 @@ def test_diff(self):
             1), 'z': pd.Series(1)}).astype('float64')
         assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize('axis', [0, 1])
+    @pytest.mark.parametrize('tz', [None, 'UTC'])
+    @pytest.mark.xfail(raises=NotImplementedError)
```

Comment:
putting the xfail here means *everything* xfails, so this is not a very useful test